src/workflows-app/SubmissionDetails.test.js

```diff
@@ -139,6 +151,14 @@ describe('Submission Details page', () => {
         Apps: {
           listAppsV2: mockLeoResponse,
         },
+        Metrics: {
+          captureEvent,
+        },
+        AzureStorage: {
+          details: jest.fn(() => {
+            return Promise.resolve({ sas: { token: '1234-this-is-a-mock-sas-token-5678' } });
+          }),
```

Comment:
Not necessary to change, but FYI, this can be written a little more concisely with [`mockResolvedValue`](https://jestjs.io/docs/mock-function-api#mockfnmockresolvedvaluevalue).

```suggestion
          details: jest.fn().mockResolvedValue({ sas: { token: '1234-this-is-a-mock-sas-token-5678' } }),
```